src/App.css

```diff
@@ -45,6 +46,9 @@
   font-size: 20px;
   padding: 10px;
   text-align: center;
+  font-family: Arial, Helvetica, sans-serif;
+  text-decoration: none;
+  color:black;
 }
 .article-img-url{
```

Comment:
the article img on mobile comes half off the page on each article page and isn't viewable!